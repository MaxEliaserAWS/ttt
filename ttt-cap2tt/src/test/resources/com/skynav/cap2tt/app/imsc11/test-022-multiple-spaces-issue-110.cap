Lambda字幕V4	DF0+1	SCENE"和文標準"

187	00130713/00130918	文文文文	＠縦右 ＠行頭
				文   文   文   文
				文___文___文___文
				文＿＿＿文＿＿＿文＿＿＿文
