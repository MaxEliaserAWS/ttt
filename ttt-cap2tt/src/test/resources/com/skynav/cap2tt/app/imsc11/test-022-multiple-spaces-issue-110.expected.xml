<?xml version="1.0" encoding="utf-8"?><tt xmlns="http://www.w3.org/ns/ttml" ttp:contentProfiles="http://www.w3.org/ns/ttml/profile/imsc1.1/text" ttp:frameRate="30" ttp:profile="http://www.w3.org/ns/ttml/profile/imsc1.1/text" ttva:model="imsc11" xml:lang="ja" xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:ttva="http://skynav.com/ns/ttv/annotations"><head><styling><initial tts:fontSize="4rh"/><initial tts:lineHeight="5rh"/><initial tts:showBackground="whenActive"/></styling><layout><region xml:id="横下" tts:displayAlign="after" tts:extent="80rw 15rh" tts:position="center bottom 10rh"/><region xml:id="縦右" tts:extent="15rw 80rh" tts:position="right 10rw center" tts:writingMode="tbrl"/></layout></head><body region="横下" xml:space="preserve"><div><p begin="00:13:07:13" end="00:13:09:18" region="縦右">文文文文<br/>文   文   文   文<br/>文   文   文   文<br/>文　　　文　　　文　　　文</p></div></body></tt>